### PR TITLE
Nick/admin dash design

### PR DESCRIPTION
**Pull request by niwong**
## Implement Basic Design of Admin Dashboard

Screen Shot 2019-11-24 at 7 02 01 PM

Describe your feature here

### Related PRs

Optional - any related PRs you're waiting on, or PRs that will conflict, etc

### Migrations

Optional - if you added anything to the database through migration(s)

### Tests Performed, Edge Cases

Lol we'll see if we test

### Screenshots

Add screenshots!!! If you'd like

CC: @aivantg

### CODE REVIEW

File: src/lib/adminHelper.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { getRecord } from './request';
+import key from './api_key';
+
+const Airtable = require('airtable');
+
+// API KEY will reside in ENV variables later.
+Airtable.configure({
```
Comment (aivantg) [inline src/lib/adminHelper.js]: Do we need to reconfigure airtable here? Not sure how the library works but I assumed this would only be done in `request.js`?

Comment (niwong) [reply src/lib/adminHelper.js]: removed for now...
